### ISSUE TITLE
Error When Installing

### ISSUE THREAD

**Issue by gigavinyl** [fisherman/fin#28]
When I attempt to install fin with fisher, I get the following error:
```
fish: Unknown command '__fin_version_local'
~/.config/fish/completions/fin.fish (line 1): __fin_version_local
                                              ^
in command substitution
	called on line -1 of file ~/.config/fish/completions/fin.fish

from sourcing file ~/.config/fish/completions/fin.fish
	called on line 719 of file ~/.config/fish/functions/fisher.fish

in function '__fisher_plugin_enable'
	called on line 392 of file ~/.config/fish/functions/fisher.fish
	with parameter list '/Users/gigavinyl/.config/fisherman/fin'

in function '__fisher_install'
	called on line 226 of file ~/.config/fish/functions/fisher.fish
	with parameter list 'fin'

in function 'fisher'
	called on standard input
	with parameter list 'fin'

Done in 447ms
```

**Comment by robsonpeixoto** [comment]
@gigavinyl Check if the file `~/.config/fish/config.fish` has the content 
```
for file in ~/.config/fish/conf.d/*.fish
    source $file
end
```

**Comment by gigavinyl** [comment]
@robsonpeixoto Ahhh thanks that got it working 😄

**Comment by bucaran** [comment]
@gigavinyl What @robsonpeixoto said will fix this only as a side-effect, actually this is a bug in fisherman. I have published fisherman 2.6.17 fixing this.

https://github.com/fisherman/fisherman/commit/868363891924af481961b67acdcb79b5817afbd8

**Comment by bucaran** [comment]
Closing as fixed. 🍻
